questionpy_sdk/webserver/form_storage.py

```diff
@@ -0,0 +1,22 @@
+import json
+from pathlib import Path
+from typing import Optional
+
+
+class FormStorage:
+
+    def __init__(self) -> None:
+        # Mapping of package path to form data path
+        self.paths: dict[Path, Path] = {}
+        self.storage_path: Path = Path(__file__).parent / 'form_storage'
+
+    def insert(self, key: Path, form_data: dict) -> None:
+        path = self.storage_path / key.with_suffix('.json')
+        self.paths[key] = path
+        json.dump(form_data, open(path, 'w'))
```

Comment:
Hier und in `get()` müssten noch `close()` aufgerufen werden, am besten direkt über einen Kontextmanager:

```suggestion
        with path.open('w') as file:
            json.dump(form_data, file)
```